shared/actions/chat2/index.tsx

```diff
@@ -1135,13 +1130,7 @@ function* loadMoreMessages(
   }
 
   const conversationIDKey = key
-
   const conversationID = Types.keyToConversationID(conversationIDKey)
-  if (!conversationID.length) {
```

Comment:
this is basically a noop, its a buffer and can never be null, even if that string is non hex you get a valid buffer (and it actually has length sadly). this should never fail as we check isValidConverationID above in string form anyways